charts/prometheus-cloudwatch-exporter/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "0.12.2"
+appVersion: "0.13.0"
```

Comment:
thats a breaking change which woul need a version 1.0 and some upgrading information.